ext/reflection/php_reflection.c

```diff
@@ -6195,277 +6195,6 @@ ZEND_METHOD(reflection_reference, getId)
 }
 /* }}} */
 
-/* {{{ method tables */
-static const zend_function_entry reflection_exception_functions[] = {
-	PHP_FE_END
-};
-
-static const zend_function_entry reflection_functions[] = {
-	ZEND_ME(reflection, getModifierNames, arginfo_class_Reflection_getModifierNames, ZEND_ACC_PUBLIC|ZEND_ACC_STATIC)
-	PHP_FE_END
-};
-
-static const zend_function_entry reflector_functions[] = {
-	PHP_FE_END
-};
-
-static const zend_function_entry reflection_function_abstract_functions[] = {
-	ZEND_ME(reflection, __clone, arginfo_class_ReflectionFunctionAbstract___clone, ZEND_ACC_PRIVATE|ZEND_ACC_FINAL)
```

Comment:
The problem is that all the `__clone()` methods in the reflection extension uses the internal `reflection, __clone` method. Isn't it possible to set a custom clone handler instead? Or should I just add the `__clone()` method to one of the classes and alias it in the others?